README.md

```diff
@@ -53,16 +60,39 @@ This section describes the terms used for all documentation in this repository.
 
   A particular version of a Linux distribution used to create a rootfs from.
 
-## Usage
+- dracut
+
+  A guest OS build method where the building host is used as the Base OS.
+  For more information refer to the [dracut homepage](https://dracut.wiki.kernel.org/index.php/Main_Page).
+
+## Building
 
 The top-level `Makefile` contains an example of how to use the available components.
 
-By default, components will run on the host system. However, some components
+Two build methods are available, "distro" and "dracut".
+By default, the "distro" build method is used, and this creates a rootfs using
+distro specific commands (e.g.: `debootstrap` for Debian or `yum` for CentOS).
+The "dracut" build method uses the distro-agnostic tool `dracut` to obtain the same goal.
+
+By default components are run on the host system. However, some components
 offer the ability to run from within Docker (for ease of setup) by setting the
 `USE_DOCKER=true` variable.
 
 For more detailed information, consult the documentation for a particular component.
 
+When invoking the appropriate make target as showed below, a single command is used
+to generate an initrd or an image. This is what happens in details:
+1. A rootfs is generated based on the specified target distribution.
+2. The rootfs is provisioned with Kata-specific components and configuration files.
+2. The rootfs is used as a base to generate an initrd or an image.
```

Comment:
Although it appears to render correctly this "double 2" is confusing. Please either number sequentially, or use the magic `1.` for all lines to have GitHub auto-number the bullets for you.